ami/flowchart/library/Roi.py

```diff
@@ -263,9 +263,10 @@ def to_operation(self, **kwargs):
         ex = self.values['extent x']
         oy = self.values['origin y']
         ey = self.values['extent y']
+        rotate = self.widget.rotate
 
         def func(img):
-            return img[slice(ox, ox+ex), slice(oy, oy+ey)], (ox, ex, oy, ey)
+            return np.rot90(img.T, -rotate)[slice(ox, ox+ex), slice(oy, oy+ey)], (ox, ex, oy, ey)
```

Comment:
don't we want to unrotate this array before returning it to put it back in the original before returning the array?

Reply:
I am not sure I understand what you mean? I do unrotate it here. Do you mean we should do that elsewhere?
This is working as intended on the jungfrau data in xpp. I have tested with multiple rotation settings.